syntax = "proto3";

import "google/protobuf/empty.proto";


service StatisticsManager {
  rpc GetStatistics (StatsRequest) returns (StatsReply) {}
  rpc GetFunctionNames (google.protobuf.Empty) returns (FunctionNamesReply) {}
}

message StatsRequest {
  string function_name = 1;
}

// The response message containing the greetings
message StatsReply {
  int32 counter = 1;
}

message FunctionNamesReply {
  string names = 1;
}